include/SugarFields/SugarFieldHandler.php

```diff
@@ -80,9 +80,11 @@ public static function fixupFieldType($field)
                case 'date':
                     $field = 'datetime';
                     break;
+/* #9232 user's selected column not displayed properly with link template
```

Comment:
Hey @gody01 Just asking why is this commented out? Is there still more work to be done to be done?